core/src/main/java/org/opensearch/sql/ast/expression/Function.java

```diff
@@ -23,8 +23,22 @@
 @EqualsAndHashCode(callSuper = false)
 @RequiredArgsConstructor
 public class Function extends UnresolvedExpression {
+
   private final String funcName;
   private final List<UnresolvedExpression> funcArgs;
+  private final Boolean isTableFunction;
```

Comment:
Is this required as there is already `TableFunction` node outside?

Reply:
Tablefunction is Unresolved Planning node which contains a function expression.  Let me rethink if I can take these parameters to TableFunction and seperate the two concepts together.